src/protocols/bitcoin/views/AccountDetails.vue

```diff
@@ -23,20 +31,51 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { computed, defineComponent } from 'vue';
 import { PROTOCOL_VIEW_ACCOUNT_DETAILS } from '@/constants';
 
 import AccountDetailsBase from '@/popup/components/AccountDetailsBase.vue';
-import BtnBox from '@/popup/components/buttons/BtnBox.vue';
+import AccountInfo from '@/popup/components/AccountInfo.vue';
+import { useAccounts, useBalances } from '@/composables';
+import { useStore } from 'vuex';
+import BalanceInfo from '@/popup/components/BalanceInfo.vue';
+import { useRoute, useRouter } from 'vue-router';
+import { ROUTE_ACCOUNT_DETAILS_TRANSACTIONS } from '@/popup/router/routeNames';
+import OpenTransferReceiveModalButton from '@/popup/components/OpenTransferReceiveModalButton.vue';
+import OpenTransferSendModalButton from '@/popup/components/OpenTransferSendModalButton.vue';
 
 export default defineComponent({
   name: PROTOCOL_VIEW_ACCOUNT_DETAILS,
   components: {
+    OpenTransferSendModalButton,
+    OpenTransferReceiveModalButton,
+    BalanceInfo,
+    AccountInfo,
     AccountDetailsBase,
-    BtnBox,
   },
+
   setup() {
+    const store = useStore();
+    const {
+      activeAccount,
+    } = useAccounts({ store });
+    const route = useRoute();
+    const router = useRouter();
+
+    const { balance } = useBalances({ store });
+
+    const balanceNumeric = computed(() => balance.value.toNumber());
+
+    (() => {
+      // BTC have only one view in ACCOUNT DETAILS
+      if (route.name !== ROUTE_ACCOUNT_DETAILS_TRANSACTIONS) {
+        router.push({ name: ROUTE_ACCOUNT_DETAILS_TRANSACTIONS });
+      }
+    })();
```

Comment:
Why is it needed - isn't it handled by the router?

Reply:
AccountDetails is wrapper for other routes and BTC have only one of them.